pyproject.toml

```diff
@@ -77,9 +77,9 @@ pytest = [
     "async-solipsism",
 ]
 mypy = [
-    "mypy",
-    "pandas-stubs",
-    "grpc-stubs",
+    "mypy >= 1.0.1, < 2",
+    "pandas-stubs == 1.5.3.230304",
```

Comment:
FYI We don't need panda dependency (finally) :D 
But it can be removed in the later PR.

Reply:
Then I'll push a separate PR to remove pandas so we can merge this

Reply:
https://github.com/frequenz-floss/frequenz-sdk-python/pull/261